draft-ietf-modpod-group-processes.md

```diff
@@ -626,8 +626,7 @@ ideal:
 The list below describes some types of disruptive behavior, but it
 is non-exhaustive.
 
-- Discussion of subjects unrelated to IETF policy, meetings,
-  activities, or technical concerns;
+- Persistent discussion of subjects unrelated to a group's charter;
```

Comment:
This is not only for (working) groups, so we need to be a bit more general; maybe

```suggestion
- Discussion of subjects unrelated to a forum's charter or scope;
```